blocks/library/post-title/index.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { registerBlockType, source } from '../../api';
+import PostTitleEdit from './post-title-edit';
+
+registerBlockType( 'core/post-title', {
+	title: __( 'Post Title' ),
+
+	keywords: [ __( 'title' ) ],
```

Comment:
This would be redundant, given that searching "title" would also match the "Post Title" title property, unless it was localized significantly different between "title" and "Post Title".